bft/testers/snowflake/runner.py

```diff
@@ -0,0 +1,184 @@
+import datetime
+import math
+import os
+import yaml
+from typing import Dict, NamedTuple
+
+from snowflake.connector import connect
+from snowflake.connector.errors import Error
+
+from bft.cases.runner import SqlCaseResult, SqlCaseRunner
+from bft.cases.types import Case
+from bft.dialects.types import SqlMapping
+
+type_map = {
+    "i8": "TINYINT",
```

Comment:
Aren't most of these shorter numeric just synonyms for decimal(38,0)? If so I suspect we shouldn't be binding them to smaller types. For example, I think i32+i32 output decimal(38,0) in snowflake as opposed to what is likely defined in substrait where i32 + i32 => i32.